### DISCUSSION TITLE
Automate publishing new releases to Marketplace

### DISCUSSION THREAD

**Discussion by mrienstra** [community/community#52135] [Actions]
### Select Topic Area

Product Feedback

### Body

This is a continuation of https://github.com/orgs/community/discussions/26410, which was marked as "answered" in 2020, and therefore is probably "off radar".

Right now, publishing a new Actions release to the Marketplace requires a manual step, which is likely a bottleneck for many projects.

The intention of this discussion:
1. Status updates: this was on the backlog in 2020 -- is that still the case?
2. To solicit input re: the need for this feature, and what is regarded as essential.
3. To find out if anything needed from the user base to move this forward.

Thanks!

**Comment by abhishek-ws** [comment]
+ 1
